{"name": "test_login_fail[fail_data3]", "status": "passed", "description": "\n        用例描述：测试登录失败用例\n        ", "attachments": [{"name": "log", "source": "7e2c9762-9ef8-4011-b2dd-04fc3fabf327-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "6ec621e4-d600-4055-b9c8-8bc2b0837468-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "fail_data", "value": "{'sUserName': '16533907260', 'sPassword': ''}"}], "start": 1586410348899, "stop": 1586410349587, "uuid": "8b95fe2e-5515-411e-9b60-63ba236fae0c", "historyId": "db57e921a10ef9659acaa552a08558fa", "testCaseId": "8000cb86548fa2cebd0d986e95a5b795", "fullName": "test_case.test_login.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录失败用例"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "5888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}